### ISSUE TITLE
trouble importing project into IDE

### ISSUE THREAD

**Issue by jb08** [scalatra/scalatra#773]
received this error when importing project into IntelliJ with `sbt version=1.0.3`:

```
Error:Error while importing SBT project:<br/>...<br/><pre>[error] 	at sbt.MainLoop$.runWithNewLog(MainLoop.scala:70)
[error] 	at sbt.MainLoop$.runAndClearLast(MainLoop.scala:52)
[error] 	at xsbt.boot.Boot.main(Boot.scala)
[error] java.lang.ClassNotFoundException: org.jetbrains.sbt.CreateTasks$
[error] Use 'last' for the full log.
[error] Not a valid key: dump-structure (similar: buildStructure)
[error] */*:dump-structure
[error]                   ^</pre><br/>See complete log in <a JIdea2016.3/sbt.last.log</a>
```

**Comment by takezoe** [comment]
What's your IntelliJ version?

**Comment by jb08** [comment]
```
IntelliJ IDEA 2016.3.5
Build #IU-163.13906.18, built on March 6, 2017
JRE: 1.8.0_112-release-408-b6 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
```

**Comment by takezoe** [comment]
That version doesn't support sbt 1.0. Please upgrade to IntelliJ 2017.2 at least.
https://blog.jetbrains.com/scala/2017/07/19/intellij-idea-scala-plugin-2017-2-sbt-1-0-improved-sbt-shell-play-2-6-and-better-implicits-management/

**Comment by jb08** [comment]
yes, IntelliJ 2017.2.5 solved it via integration with sbt 1.0+ plugin. no more squiggly lines, thanks @takezoe
![screen shot 2017-12-08 at 1 19 20 pm](https://user-images.githubusercontent.com/8146460/33781534-9d1fab58-dc1a-11e7-89b1-7e08e08a3ee2.png)

**Comment by takezoe** [comment]
👍
